packages/core/admin/shared/contracts/admin.ts

```diff
@@ -1,4 +1,71 @@
 import { errors } from '@strapi/utils';
+import { ValidationError as YupValidationError } from 'yup';
+
+// TODO replace SettingsFile with type from the upload plugin ?
+interface Logo {
+  name: string;
+  url: string;
+  width: number;
+  height: number;
+  ext: string;
+  size: number;
+}
+
+export type SettingsFile = any;
+
+/**
+ * /init - Initialize the admin panel
+ */
+export declare namespace Init {
+  export interface Request {
+    body: {};
+    query: {};
+  }
+  export interface Response {
+    data: {
+      uuid: string | false;
+      hasAdmin: boolean;
+      menuLogo?: SettingsFile;
+      authLogo?: SettingsFile;
```

Comment:
I think you always send an object or Logo for these, same with `project-settings`